test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -0,0 +1,235 @@
+            var examplePathEmptyFile = Path.Combine(UnitTestBase.TestDirectory, @"core\Home.dyn");
+
+            // Act
+            // open file
+            Open(examplePathIronPython);
+            var ironPythonWorkspaceId = this.ViewModel.CurrentSpace.Guid;
+            var ironPythonDialog = GetOpenWindowsOfType<IronPythonInfoDialog>(this.View).First();
+            Assert.IsNotNull(ironPythonDialog);
+            ironPythonDialog.OkBtn.RaiseEvent(new RoutedEventArgs(Button.ClickEvent));
+
+            // Open empty file before open the the IronPython file again
+            Open(examplePathEmptyFile);
+            Assert.AreNotEqual(ironPythonWorkspaceId, this.ViewModel.CurrentSpace.Guid);
+            
+
+            Open(examplePathIronPython);
+            Assert.AreEqual(ironPythonWorkspaceId, this.ViewModel.CurrentSpace.Guid);
+            var secondGraphIronPythonDialog = GetOpenWindowsOfType<IronPythonInfoDialog>(this.View);
+
+            // Assert
+            Assert.AreEqual(0, secondGraphIronPythonDialog.Count());
+        }
+
+        /// <summary>
+        /// This tests checks if the extension can detect IronPython nodes in the graph 
+        /// </summary>
+        [Test]
+        public void CanDetectIronPythonNodesInGraph()
+        {
+            // Arrange
+            RaiseLoadedEvent(this.View);
+            var extensionManager = View.viewExtensionManager;
+            extensionManager.Add(viewExtension);
+
+            // Act
+            // open file
+            Open(@"core\python\python.dyn");
+
+            var pythonMigration = extensionManager.ViewExtensions.Where(x => x.Name == viewExtension.Name).Select(x=>x).First() as PythonMigrationViewExtension;
+
+            // Assert
+            Assert.IsTrue(pythonMigration.PythonDependencies.ContainsIronPythonDependencies());
+        }
+
+        /// <summary>
+        /// Checks if pressing the `More Information` button on the IronPythonInfoDialog window
+        /// will open the DocumentaionBrowser viewExtension.
+        /// </summary>
+        [Test]
+        public void CanOpenDocumentationBrowserWhenMoreInformationIsClicked()
+        {
+            // Arrange
+            RaiseLoadedEvent(this.View);
+            var extensionManager = View.viewExtensionManager;
+            extensionManager.Add(viewExtension);
+
+            // Act
+            // open file
+            var examplePath = Path.Combine(UnitTestBase.TestDirectory, @"core\python", "python.dyn");
+            Open(examplePath);
+
+            var ironPythonDialog = GetOpenWindowsOfType<IronPythonInfoDialog>(this.View).First();
+            var viewExtensionTabsBeforeBtnClick = this.View.ExtensionTabItems.Count;
+
+            ironPythonDialog.MoreInformationBtn.RaiseEvent(new RoutedEventArgs(Button.ClickEvent));
+
+            // Assert
+            Assert.AreNotEqual(viewExtensionTabsBeforeBtnClick, this.View.ExtensionTabItems.Count);
+            Assert.IsTrue(this.View.ExtensionTabItems
```

Comment:
move `this.View.ExtensionTabItems.Any(x => x.Header.ToString() == "Documentation Browser"` to a `hasDocumentationBrowserTab` variable above